testsuites/listener/shared/client_sg/test_replication.py

```diff
@@ -1058,3 +1058,167 @@ def test_verify_open_revs_with_revs_limit_push_conflict(setup_client_syncgateway
+    2. Create a new session on SGW for the user1 by using POST /_session.
+       Capture the SyncGatewaySession cookie from the set-cookie in the response header.
+    3. Start continuous push replicator on the LiteServ with SyncGatewaySession cookie.
+       Make sure that the replicators start correctly
+       Let the replicator run for more than 10% of the session expiry time specified when the session was created in step 2
+    4. Cancel both push and pull replicator on the LiteServ
+    5. Delete the session from SGW by sending DELETE /_sessions/ to SGW
+    6. Repeat steps 2 through 6
+    """
+
+    ls_db = "ls_db"
+    sg_db = "db"
+
+    cluster_config = setup_client_syncgateway_test["cluster_config"]
+    sg_mode = setup_client_syncgateway_test["sg_mode"]
+    ls_url = setup_client_syncgateway_test["ls_url"]
+    sg_url = setup_client_syncgateway_test["sg_url"]
+    sg_admin_url = setup_client_syncgateway_test["sg_admin_url"]
+
+    sg_config = sync_gateway_config_path_for_mode("listener_tests/listener_tests_user", sg_mode)
+    c = cluster.Cluster(config=cluster_config)
+    c.reset(sg_config_path=sg_config)
+
+    log_info("Running 'test_replication_with_session_cookie_short_ttl'")
+    log_info("ls_url: {}".format(ls_url))
+    log_info("sg_admin_url: {}".format(sg_admin_url))
+    log_info("sg_url: {}".format(sg_url))
+
+    client = MobileRestClient()
+    client.create_database(url=ls_url, name=ls_db)
+
+    # Get session header for user_1
+    cookie_name, session_id = client.create_session(url=sg_admin_url, db=sg_db, name="user_1", password="foo", ttl=10)
+
+    # session_header: SyncGatewaySession=a483be3248f740d810c09eb2c1b1f9198141bb15; Path=/db; Expires=Fri, 14 Apr 2017 03:54:46 GMT
+    session_header = "{}={}".format(cookie_name, session_id)
+
+    log_info("session_header: {}".format(session_header))
+
+    # Start authenticated push replication
+    repl_one = client.start_replication(
+        url=ls_url,
+        continuous=True,
+        from_db=ls_db,
+        to_url=sg_url,
+        to_db=sg_db,
+        to_auth=session_header
+    )
+
+    # Wait for 1 replications to be 'Idle', On .NET they may not be immediately available via _active_tasks
+    client.wait_for_replication_status_idle(ls_url, repl_one)
+
+    replications = client.get_replications(ls_url)
+    assert len(replications) == 1, "1 replications (push should be running)"
+
+    num_docs_pushed = 1
+    attempts = 3
+
+    # Add 3 docs and sleep in between
+    while attempts > 0:
+        # Sanity test docs
+        ls_prefix = "ls_doc_" + str(attempts)
+        ls_docs = client.add_docs(url=ls_url, db=ls_db, number=num_docs_pushed, id_prefix=ls_prefix, channels=["ABC"])
+        assert len(ls_docs) == num_docs_pushed
+
+        attempts -= 1
+        # Sleep for 5 seconds after every add so as to expire the ttl
+        time.sleep(5)
+
+    all_docs = client.merge(ls_docs)
```

Comment:
I was originally adding docs to both SG and CBL. I removed the code to add docs to SG but retained this piece of code. Removed it